Readme.md

```diff
@@ -3,8 +3,8 @@
 <h5 align="center">Fast, flexible & lean implementation of core jQuery designed specifically for the server.</h5>
 
 <div align="center">
-  <a href="https://travis-ci.org/cheeriojs/cheerio">
-    <img src="https://img.shields.io/travis/cheeriojs/cheerio/main" alt="Travis CI">
+  <a href="https://github.com/cheeriojs/cheerio/actions?query=workflow%3ACI+branch%3Amain">
+    <img src="https://img.shields.io/github/workflow/status/cheeriojs/cheerio/CI/main" alt="Build Status">
```

Comment:
Should lint and CI be in the same file (as different jobs), to make this cover both? (That's the current behavior, but I can also see a point about having them be separate.)

Reply:
The reason I split the actions is so that everything is faster since they run in parallel this way. I doubt having a badge to lint is something that people should care about :)

Reply:
Wouldn't separate jobs be executed in parallel as well?

Reply:
Yeah, but I personally find this cleaner. Each workflow does one thing. That being said, the types test could probably be moved to the lint workflow.

Reply:
Makes sense & I can live with this.